src/gridcoin/voting/registry.h

```diff
@@ -184,6 +197,8 @@ class PollRegistry : public IContractHandler
     using PollMapByTitle = std::map<std::string, PollReference>;
     using PollMapByTxid = std::map<uint256, PollReference*>;
 
+    static CCriticalSection cs_poll_registry;  //!< Lock for poll registry.
```

Comment:
Why is this static? (even if PollRegistry is used as a singleton.)

Reply:
Ah, because originally I was experimenting with using the newer non-recursive mutex, and the compiler was complaining when it was a non-static member of the class. I decided not to go that route, so I should revert it back.